### PR TITLE
fixes issues ui issue wher images where cropped

### PR DESCRIPTION
**Pull request by irnh99**
- removes container page that takes the full page

### CODE REVIEW

File: src/app/tab2/tab2.page.html [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <ion-header [translucent]="true">
   <ion-toolbar>
-    <ion-title>
-      Photo Gallery
-    </ion-title>
+    <ion-title> Photo Gallery </ion-title>
```
Comment (DREADSKY) [inline src/app/tab2/tab2.page.html:3]: Is it a best practice to write titles between blocks, or is there more too it than just that?

File: src/app/tab2/tab2.page.html [hunk 2]
```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
-</ion-content>
-
-<ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
```
Comment (DREADSKY) [inline src/app/tab2/tab2.page.html:19]: I accidentally duplicated it…? That's why it was showing up double…

File: src/app/tab2/tab2.page.html [hunk 3]
```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
```
Comment (DREADSKY) [inline src/app/tab2/tab2.page.html:16]: Was this just unnecessary and did it add to the error?

File: src/app/tab2/tab2.page.html [hunk 4]
```diff
@@ -13,28 +11,22 @@
...
-        <ion-img [src]="photo.webviewPath"></ion-img>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
-
   <ion-content>
+    <ion-grid>
+      <ion-row>
+        <ion-col
+          size="6"
```
Comment (DREADSKY) [inline src/app/tab2/tab2.page.html:18]: I'm a little confused at what happened here? Is there a best practice that says they must be on different lines, also why the space at the end?

File: src/app/tab2/tab2.page.html [hunk 5]
```diff
@@ -13,28 +11,22 @@
...
+      </ion-row>
+    </ion-grid>
+
     <ion-fab vertical="bottom" horizontal="center" slot="fixed">
       <ion-fab-button (click)="addPhotoToGallery()">
         <ion-icon name="camera"></ion-icon>
       </ion-fab-button>
     </ion-fab>
   </ion-content>
-
```
Comment (DREADSKY) [inline src/app/tab2/tab2.page.html:39]: Too many spaces?

**Review by DREADSKY — APPROVED** [review]
Thanks for the help, I'll love to hear more feedback from you.

File: src/app/tab2/tab2.page.html (hunk 1 above)
Comment (irnh99) [reply src/app/tab2/tab2.page.html:3]: In general makes it easier to read and edit, I use a beautifier extension on vs, which basically orders the code automatically

File: src/app/tab2/tab2.page.html (hunk 2 above)
Comment (irnh99) [reply src/app/tab2/tab2.page.html:19]: Yeah is that and the class one element had which basically is a full screen element

File: src/app/tab2/tab2.page.html (hunk 3 above)
Comment (irnh99) [reply src/app/tab2/tab2.page.html:16]: Well the explore container is another component that you where invoking, basically you where calling that page to be inside there, but you didn't have any reason to implement that component

File: src/app/tab2/tab2.page.html (hunk 4 above)
Comment (irnh99) [reply src/app/tab2/tab2.page.html:18]: Its mostly a rule of how easy is to read and to edit, I use a beautifier tool to do that automatically, usually it is better to use different lines. That being said the rules is really just based on how long a line is and if the line would over flow most screens.

File: src/app/tab2/tab2.page.html (hunk 5 above)
Comment (irnh99) [reply src/app/tab2/tab2.page.html:39]: Well usually in html there is no empty lines you can add them back if you feel it makes it easier to read

File: src/app/tab2/tab2.page.html (hunk 1 above)
Comment (DREADSKY) [reply src/app/tab2/tab2.page.html:3]: I'll add the beautifier extension then, might make things easier for me.

File: src/app/tab2/tab2.page.html (hunk 2 above)
Comment (DREADSKY) [reply src/app/tab2/tab2.page.html:19]: Class one? I'll have to refresh my memory on that, but basically it's an element outside a code block, right?

File: src/app/tab2/tab2.page.html (hunk 4 above)
Comment (DREADSKY) [reply src/app/tab2/tab2.page.html:18]: Oh, I get what you're saying, my screen is basically a TV in my room, so it doesn't usually overflow, I shall keep that in mind.

File: src/app/tab2/tab2.page.html (hunk 5 above)
Comment (DREADSKY) [reply src/app/tab2/tab2.page.html:39]: Got ya! I'll remember that, readability is important, and I'll probably need to conform to a reading style that's best for everyone if I am to work on T3.

**Pull request merged**